Library/GeneralImage.cpp

```diff
@@ -296,6 +299,76 @@ bool GeneralImage::LoadImage(const std::wstring& imageName)
 	return false;
 }
 
+bool GeneralImage::LoadImageFromMeasure(Measure* m)
+{
+	if (!m_Skin || m == nullptr)
+	{
+		DisposeImage();
+		return false;
+	}
+
+	auto pm = dynamic_cast<MeasurePlugin*>(m);
+	if (pm == nullptr)
+	{
+		DisposeImage();
+		return false;
+	}
+
+	ImageOptions info;
+	info.m_Path = std::wstring{ L"measure://" } + m->GetSkin()->GetFilePath() + L"/" + m->GetName();

```

Comment:
Btw, is this what will be displayed in log window?

Reply:
I don't think we should display anything related to images in the `About` window.
1. It would be hard to implement.
2. You could say that images are somewhat like a new section variable. There could possibly be infinite section variables in any plugin, and rainmeter displays none of them. Well, nothing bad will happen if it will also not display images.

Reply:
Oh, since i saw `measure://` i thought that what would be displayed.
Okay, in normal case (without implementation) what is gonna be displayed in about page? an empty string?

Reply:
Nah, that's just a cache key, that must be unique for cache to work properly.

> Okay, in normal case (without implementation) what is gonna be displayed in about page? an empty string?

What do you mean "without implementation"?
As I see it, image will be just the third measure value. Number and string values will not be affected.
I think that any mentions of images in the `About` window would be just unneeded work, because they do nothing for the purpose of transferring. If everything works, _and_ rainmeter devs accept the proposal, only then you could _start_ thinking about adding info about images into about window.

Reply:
>What do you mean "without implementation"?

👇

>>I don't think we should display anything related to images in the About window.
>>1. It would be hard to implement.

>If everything works, and rainmeter devs accept the proposal, only then you could start thinking about adding info about images into about window.

Yeah i'll worry about it later, but i wanna implement it before making the proposal, i don't want them to reject it for something simple like this.

Reply:
I think that reject with a reason "Add some info into `About` window and it will be OK" is very different from "We don't want this at all, we will never accept it". And simply missing the info in GUI won't cause the second type of reject.